### ISSUE TITLE
[BUG]: Unable to install Android app after creation of the project

### ISSUE THREAD

**Issue by imAunAbbas** [thecodingmachine/react-native-boilerplate#339]
### react-native environment setup

- [X] I verified that I have not react-native-cli installed globally and be up to date about the React Native environment setup.

### Description

I've recently installed the latest version of this boilerplate for my new project. After the installation, iOS application is working fine, but Android application is not installing on the emulator. I've checked my development environment as well, as my other projects (including the new one) are working.
When I try to run the project on Android, it gives the following error:

`The Kotlin Gradle plugin was loaded multiple times in different subprojects, which is not supported and may break the build. 
This might happen in subprojects that apply the Kotlin plugins with the Gradle 'plugins { ... }' DSL if they specify explicit versions, even if the versions are equal.
Please add the Kotlin plugin to the common parent project or the root project, then remove the versions in the subprojects.
If the parent project does not need the plugin, add 'apply false' to the plugin line.
See: https://docs.gradle.org/current/userguide/plugins.html#sec:subprojects_plugins_dsl
The Kotlin plugin was loaded in the following projects: ':react-native-gesture-handler', ':react-native-screens'

> Task :app:processDebugResources FAILED
106 actionable tasks: 10 executed, 96 up-to-date

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:processDebugResources'.
> A failure occurred while executing com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask$TaskAction
   > Android resource linking failed
     /Users/<Name>/Desktop/repo/Project/android/app/build/intermediates/packaged_manifests/debug/AndroidManifest.xml:14: error: unexpected element <adaptive-icon> found in <manifest>.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
java.lang.StackOverflowError (no error message)

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 13s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:processDebugResources'.
> A failure occurred while executing com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask$TaskAction
   > Android resource linking failed
     /Users/<name>/Desktop/repo/Project/android/app/build/intermediates/packaged_manifests/debug/AndroidManifest.xml:14: error: unexpected element <adaptive-icon> found in <manifest>.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
java.lang.StackOverflowError (no error message)

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 13s

    at makeError (/Users/<Name>/Desktop/repo/Project/node_modules/execa/index.js:174:9)
    at /Users/<Name>/Desktop/repo/Project/node_modules/execa/index.js:278:16
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async runOnAllDevices (/Users/<Name>/Desktop/repo/Project/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/runOnAllDevices.js:82:7)
    at async Command.handleAction (/Users/<Name>/Desktop/repo/Project/node_modules/@react-native-community/cli/build/index.js:108:9)
info Run CLI with --verbose flag for more details.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

### Version

3.6.4

### On which OS this issue appear on?

- [ ] IOS
- [X] Android

### Desktop OS

macOS Ventura 13.2

**Comment by JeremyDolle** [comment]
Hi @imAunAbbas 

I am on it ! If you want to fix it quickly remove the following in the AndroidManifest file : 
```
<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
      <background android:drawable="@color/ic_launcher_background" />
      <foreground android:drawable="@mipmap/ic_launcher_foreground" />
    </adaptive-icon>
```

Sorry for that it will be fixed in the next release

**Comment by imAunAbbas** [comment]
Hi @JeremyDolle 

Thank you for the quick response. Can you please tell me how much time will it take for the next release?

**Comment by JeremyDolle** [comment]
Sorry but I don't know the exact date of the release now because there is also a graphic overhaul in progress and the fix is planned on the same sprint.

Hope the next week or the next after we can provide a release of all the in progress work

**Comment by JeremyDolle** [comment]
Fixed in [v3.6.5](https://github.com/thecodingmachine/react-native-boilerplate/releases/tag/v3.6.5). Enjoy !
